instance/server/Router.elm

```diff
@@ -86,27 +87,34 @@ routePost ( req, res ) model =
                 |> runRouteWithErrorHandler
             )
     in
-        if url == routes.apply then
-            model
-                => generate generateSuccessPage
-        else if url == routes.signup then
-            model
-                => generate generateSignupPage
-        else if url == routes.startTest then
-            model
-                => generate generateTestPage
-        else if url == routes.login then
-            model
-                => generate generateAdminPage
-        else if url == routes.registerUser then
-            model
-                => generate generateSuccessfulRegistrationPage
-        else
-            model
-                => (handleError res (Task.fail "Route not found")
-                        |> runRouteWithErrorHandler
-                   )
-
+        case possibleRoute of
```

Comment:
yes, it could be: `match routes req.url |> Maybe.withDefault (NotFound req.url)`